app/scripts/einkaufsrechnungen/detail/bestellung.html

```diff
@@ -22,7 +22,7 @@ <h3 class="panel-title"><span class="navbar-brand noLeftSpace">{{sammelbestellun
               {{position.produktBeschrieb}}
             </td>
             <td ng-show="projekt.preiseSichtbar">
-              {{projekt.waehrung}} {{position.preis | ooCHF}}
+              {{position.preis | ooCurrency:projekt.waehrung:true}}
```

Comment:
Instead of feeding the `projekt.waehrung` every time to the filter, wouldn't it be possible to fetch it right in the filter-controller? Maybe getting it from $rootScope (to not load it every time from remote resource)

Reply:
done